git-link.el

```diff
@@ -285,9 +286,14 @@ return (FILENAME . REVISION) otherwise nil."
       (cond
        ((eq major-mode 'dired-mode)
         (setq filename (dired-file-name-at-point)))
-       ((and (string-match-p "^magit-" (symbol-name major-mode))
-             (fboundp 'magit-file-at-point))
-        (setq filename (magit-file-at-point)))))
+       ((or
+         (git-link--using-magit-blame)
+         (string-match-p "^magit-" (symbol-name major-mode)))
+        (setq filename (or
+                        (and (fboundp 'magit-file-at-point)
```

Comment:
If we're in git blame mode`(fboundp 'magit-file-at-point)` will always be `t` but it will always return `nil`, right?  So it should only be executed when `(string-match-p "^magit-" (symbol-name major-mode))` is `t`. No need to always evaluate it